Steps/InstallCoverageTools.yml

```diff
@@ -8,7 +8,8 @@
 steps:
 
 - powershell: |
-    choco install opencppcoverage
+    Invoke-WebRequest -Uri https://github.com/OpenCppCoverage/OpenCppCoverage/releases/download/release-0.9.9.0/OpenCppCoverageSetup-x64-0.9.9.0.exe -OutFile $(Agent.TempDirectory)\OpenCppCoverageInstall.exe

```

Comment:
I was going to say it would be nice to have a way to better manage versioning (i.e. knowing when new versions are available), but given this release was almost 3 years ago, might not be a big priority.